prov/rxm/src/rxm_init.c

```diff
@@ -191,7 +191,7 @@ static int rxm_init_info(void)
 	size_t param;
 
 	if (!fi_param_get_size_t(&rxm_prov, "buffer_size", &param)) {
-		if (param > sizeof(struct rxm_pkt)) {
+		if (param > (sizeof(struct rxm_pkt)+ sizeof(struct rxm_rndv_hdr))) {
```

Comment:
The way this if statement is structured made the change look wrong to me at first glance.  It looks like the eager_limit is somehow related to the rndv_hdr size, which it's not.  Can you change the code to this?
```
if (param <= sizeof(pkt) + sizeof(rndv_hdr)) {
    FI_WARN(...)
    return FI_EINVAL;
}

eager_limit = param - sizeof(pkt);
```
I want to make it clearer that the if statement is only validating the minimal buffer size, and that setting the eager_limit is just the success case.

Reply:
Done. I did change it so that FI_OFI_RXM_BUFFER_SIZE can be 168 bytes instead of 169 bytes.